pkg/proxy/iptables_portalmanager.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
```

Comment:
I wasn't sure about that, I'm not a google employee but I have signed the CLA, so I just copied the header from proxier, should I go in and change both new files to 2015 or (?)